### PR TITLE
feature(newUser):Loginform & RegForm added)

### PR DESCRIPTION
**Pull request by BlesyMathew**
Added login form and Registration form using shadcn ,Tailwind
image
image

### CODE REVIEW

File: src/components/layouts/fullLayout.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Outlet } from "react-router-dom";
 
 const FullLayout = (): JSX.Element => {
   return (
-    <div style={{ background: "black" }}>
+    <div style={{ background: "#a58a8a" }}>
```
Comment (shyluthomas) [inline src/components/layouts/fullLayout.tsx]: Do not use inline style: add sass for this instead

File: src/pages/user/newUser.tsx [hunk 2]
```diff
@@ -1,5 +1,215 @@
...
+export default function NewUser() {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm();
+
+  const submitLogin = (data: any) => {
+    console.log("data", data);
```
Comment (shyluthomas) [inline src/pages/user/newUser.tsx]: remove console

File: src/pages/user/newUser.tsx [hunk 3]
```diff
@@ -1,5 +1,215 @@
...
+                  id="username"
+                  placeholder="User Name"
+                  {...register("username", { required: true })}
+                />
+                {errors.username && errors.username.type === "required" && (
+                  <p className="text-xs text-rose-500 text-left">
+                    User Name is required
+                  </p>
+                )}
+              </div>
```
Comment (shyluthomas) [inline src/pages/user/newUser.tsx:101]: just add confirm password logic too

**Review by shyluthomas — CHANGES_REQUESTED** [review]

File: src/components/layouts/fullLayout.tsx (hunk 1 above)
Comment (BlesyMathew) [reply src/components/layouts/fullLayout.tsx]: ok used tail wind

File: src/pages/user/newUser.tsx (hunk 2 above)
Comment (BlesyMathew) [reply src/pages/user/newUser.tsx]: done

File: src/pages/user/newUser.tsx (hunk 3 above)
Comment (BlesyMathew) [reply src/pages/user/newUser.tsx:101]: Added validation and confirm password logic using yup
Thanks :)

**Review by shyluthomas — APPROVED** [review]

**Pull request merged**